models/vrp.rb

```diff
@@ -694,6 +704,88 @@ def calculate_service_exclusion_costs(type = :time, force_recalc = false)
       nil
     end
 
+    def service_reachable_by_any_vehicle(service)
```

Comment:
Are these change required by the introduction of same_vehicle ?

If so, i you prefer to put them in a brand new class or module instead of just moving them into vrp.rb.

Any idea of the design pattern we could apply here ?

Reply:
In fact this needs to be improved with : https://gitlab.com/mapotempo/optimizer-api/-/issues/817

Reply:
Right, I could first move `service_reachable_by_vehicle` and `service_uncompatible_with_vehicle_timewindows` to service.rb.

Do you have any further suggestions ? @senhalil ?

Reply:
Discussed during daily : we move those to wrapper.rb. In the later this will edit service.compatible_vehicles